MagmaSafe.Repositories/SQLStatements/UserStatements.cs

```diff
@@ -8,7 +8,7 @@ public class UserStatements
             INSERT INTO 
                 User
             VALUES (
-                @Id, @Name, @Email, @Password, @IsAdmin, @IsActive
+                @Id, @Name, @Email, '', @IsAdmin, @IsActive
```

Comment:
Remover completamente

```suggestion
                @Id, @Name, @Email, @IsAdmin, @IsActive
```

Reply:
Alteração efetuada